piton/design/include/piton_include.core

```diff
@@ -5,10 +5,11 @@ description: Top-level Piton includes
 filesets:
     rtl:
         files:
-            - piton_system.vh: {is_include_file: true}
+            - iop.h: {is_includ_file: true}
             - network_define.v: {is_includ_file: true}
+            - piton_system.vh: {is_include_file: true}
             - sys.h: {is_includ_file: true}
-            - iop.h: {is_includ_file: true}
+            - tlu.h: {is_includ_file: true}
         depend:
```

Comment:
Typos here. Missing an `e` in `include`. Couldn't figure out if this was addressed in a later commit

Reply:
Thanks Olof, I just pushed the commit ( c0aea52a6de05bdd86db2e673e94c57642b81a25 ) that fixes the typos.